templates/nginx/service.yaml

```diff
@@ -13,6 +13,9 @@ metadata:
 {{- end }}
 spec:
   type: ClusterIP
+  {{- if .Values.expose.clusterIP.staticClusterIP }}
+  clusterIP: {{ .Values.expose.clusterIP.clusterIP }}
```

Comment:
line 17. it should also be `.Values.expose.clusterIP.staticClusterIP` I suppose.

Reply:
You're totally right, Fixed.